CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
 
 ### Added
 
+- Rename `Call` to `Call16` and `Call17` to `Call` (#125)
```

Comment:
We definitely should add a big bold **Breaking:** here because it'll break all code with `Call`s: both for 1.6 and post-1.6 users.

Reply:
Done.